### PR TITLE
Breaking: Remove shortcuts

### PR DESCRIPTION
**Pull request by sarvaje**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [x] Added/Updated related tests.

### CODE REVIEW

File: packages/extension-browser/src/content-script/connector.ts [hunk 1]
```diff
@@ -10,13 +10,12 @@ import {
...
     traverse
 } from '@hint/utils-dom';
-
 import {
-    ConnectorOptionsConfig,
     IConnector,
     FetchEnd,
     NetworkData
 } from 'hint/dist/src/lib/types';
+import { ConnectorOptionsConfig } from '@hint/utils/dist/src/types/config';
```
Comment (molant) [inline packages/extension-browser/src/content-script/connector.ts]: `/dist/src/types/...`  😞

Can we make the types a top level of the packages? Is this part of one of the pending `utils` issues we have opened?

Comment (sarvaje) [reply packages/extension-browser/src/content-script/connector.ts]: The are at the top level of the packages, but this is the extension, and the utils package, so we need to use the full path.

Comment (molant) [reply packages/extension-browser/src/content-script/connector.ts]: I believe that in talking with @antross we were OK to link to the root path now that everything was split even if that was a small bump on the extension size.

Can you give it a try and see how much bigger it becomes?

Comment (sarvaje) [reply packages/extension-browser/src/content-script/connector.ts]: I didn't try lately, but the error we use to have using `@hint/utils`  wasn't only the size, it was that the build of the extension fails. But I will try again

Comment (sarvaje) [reply packages/extension-browser/src/content-script/connector.ts]: we can use `@hint/utils` only if we are importing types, but if we are importing a method, then we need to use the path to the file.

**Comment by sarvaje** [comment]
@molant, can we merge this?

**Comment by molant** [comment]
Yes, can you take care of it? I won't have access to my laptop until later today


________________________________

**Pull request merged**